### PR TITLE
Update Dockerfile with options for catalog names

### PR DESCRIPTION
**Pull request by grahampugh**
Added defaults for catalog names, and options flags to the python command.
My fork git clones the forked python script - if you chose to merge, watch out for that line!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,11 +6,14 @@ ENV DOCKER_TRELLO_TO_DEV_LIST="To Development" \
...
+    DOCKER_TRELLO_SUFFIX="Production" \
+    DOCKER_DEV_CATALOG="development" \
+    DOCKER_TEST_CATALOG="testing" \
+    DOCKER_PROD_CATALOG="production"
 RUN apt-get update && \
     apt-get install -y git && \
     apt-get install -y python-pip && \ 
     pip install trello && \
-    git clone https://github.com/pebbleit/munki-trello.git /munki-trello && \
+    git clone https://github.com/grahampugh/munki-trello.git /munki-trello && \
```
Comment (grahamgilbert) [inline Dockerfile]: This needs to point back to pebble's repo

**Comment by grahamgilbert** [comment]
Happy to merge once the line I highlighted has been sorted.

**Comment by grahampugh** [comment]
Done.

Cheers, Graham

Dr Graham R PughIT Support Specialist

University of Bristol / Zone D

Subject Specialist - Maths

Mac System Administration Specialist

**Comment by grahamgilbert** [comment]
Did you push your changes? Not seeing it.

**Comment by grahampugh** [comment]
Sorry, had to change it back so I could use it. I’ll change it again now.

Dr Graham R PughIT Support Specialist

University of Bristol / Zone D

Subject Specialist - Maths

Mac System Administration Specialist

**Comment by grahampugh** [comment]
Done now.

Dr Graham R PughIT Support Specialist

University of Bristol / Zone D

Subject Specialist - Maths

Mac System Administration Specialist

**Comment by grahamgilbert** [comment]
Merged and the docker image should be building now.

**Pull request merged**